typing/printtyp.ml

```diff
@@ -2176,15 +2176,14 @@ let prepare_trace f tr =
 (** Keep elements that are not [Diff _ ] and take the decision
```

Comment:
My comment might seem off-topic, but since you're already cleaning that area: did I misread/understand, or is that documentation incorrect?

Reply:
The `not` is a typo indeed.